### DISCUSSION TITLE
Type Error

### DISCUSSION THREAD

**Discussion by j3nsykes** [FlowiseAI/Flowise#1059] [Q&A]
I'm  getting a `TypeError: Cannot read properties of undefined (reading 'replace')`
with a simple Q and A chain from a text file.  This is following the video tutorials provided so not sure what the issue is. I see in other discussions people deleted nodes and started again but this does not resolve the issue in this instance. 

<img width="1489" alt="Screenshot 2023-10-14 at 17 42 57" src="https://github.com/FlowiseAI/Flowise/assets/31961310/10f2a222-0584-474e-870d-83da8c5914b6">

**Comment by j3nsykes** [comment]
resolved by starting a new chatflow from scratch. Would like to know a more efficient way to resolve this error though

**Reply by aiaisir66** [reply]
I have the same ERROR and no other ideal than starting again from scratch...

**Reply by mkmatzat** [reply]
I deployed 1.6.3 on a Railway server and have a similar error with my flow.
Sometimes it works the first query after redeploying. But then I get:

_TypeError: (intermediate value) is not iterable (cannot read property undefined)_

always. 

On my local Flowise installation on a Windows Server and the excact flow, I never saw this problem.
